src/plugin/file/shp/ui/zipshpimportui.js

```diff
@@ -63,9 +67,21 @@ plugin.file.shp.ui.ZipSHPImportUI.prototype.launchUI = function(file, opt_config
     this.config_ = opt_config;
   }
 
-  var content = /** @type {ArrayBuffer} */ (file.getContent());
-  zip.createReader(new zip.ArrayBufferReader(content),
-      this.handleZipReader.bind(this), this.handleZipReaderError.bind(this));
+  const content = /** @type {ArrayBuffer|Blob} */ (file.getContent());
+
+  if (content instanceof ArrayBuffer) {
+    zip.createReader(new zip.ArrayBufferReader(content),
+        this.handleZipReader.bind(this), this.handleZipReaderError.bind(this));
+  } else if (content instanceof ArrayBuffer) {
```

Comment:
This condition is the same as the `if`.

Reply:
Woops, meant to explicitly check `Blob` here.